modules/services/backup/restic.nix

```diff
@@ -0,0 +1,52 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+{
+  options = {
+    backup = mkOption {
+      type = types.listOf types.string;
+      default = [ ];
+      example = [ "/path/to/backup/directory" ];
+      description = "Directories to be backed up using restic";
+    };
+
+    resticPasswordFile = mkOption {
+      type = types.string;
+      default = "";
```

Comment:
Don't default to `""`, default to `null`. That way, if the file is not set, it will throw an error.